crates/forge/src/test_execution_syscall_handler.rs

```diff
@@ -635,25 +637,24 @@ struct ScarbStarknetContractArtifact {
 }
 
 fn execute_syscall(
-    system: &ResOperand,
     vm: &mut VirtualMachine,
     exec_scopes: &mut ExecutionScopes,
     hint_data: &Box<dyn Any>,
     constants: &HashMap<String, Felt252>,
     cheatable_syscall_handler: &mut CheatableSyscallHandler,
 ) -> Result<(), HintError> {
-    let (cell, offset) = extract_buffer(system);
-    let system_ptr = get_ptr(vm, cell, &offset)?;
```

Comment:
Can we execute `validate_syscall_ptr` here? to make sure `system_ptr` and `cheatable_syscall_handler.syscall_handler.syscall_ptr` are equal?